blueprints/nucleus-addons/files/packages/__name__/addon/styles/addon.scss

```diff
@@ -0,0 +1,3 @@
+@import "nucleus/variables";
+@import "nucleus/animations";
```

Comment:
Let's not have animations as a part of the blueprint, as many components might not need them.

Reply:
Removed animations.